delta/extensions/sources/sentinel1.py

```diff
@@ -0,0 +1,149 @@
+# Copyright © 2020, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All rights reserved.
+#
+# The DELTA (Deep Earth Learning, Tools, and Analysis) platform is
+# licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Functions to support Sentinel1 satellites.
+"""
+
+import os
+import sys
+import portalocker
+
+import tensorflow as tf
```

Comment:
Your pytest error is from this (and the same in worldview file), you can just use normal print here. Also merge in my latest changes, it fixes another build error we were having from an h5py update.